### PR TITLE
Adds Swedish translations 

### PR DESCRIPTION
**Pull request by njons**
Summary: 
- adds Swedish translation to content in dashboard and signup 
- fixes a failing render test (following content change)
- sets Travis.yml node version to v10 (although v10 should should remain LTS until April 2020, LTS is v12 on Travis and node-sass becomes incompatible)

### CODE REVIEW

File: i18n/src/i18n-messages.json [hunk 1]
```diff
@@ -441,7 +445,7 @@
   },
   {
     "id": "verify-identity.vetting_freja_tagline",
-    "defaultMessage": "For those with the app able to pass by one of the authorised agents"
+    "defaultMessage": "For those able to create a Freja eID by passing by one of the authorised agents"
```
Comment (enriquepablo) [inline i18n/src/i18n-messages.json]: I think that the 2nd `by` should be removed?
If I understand correctly ;-)

File: i18n/src/i18n-messages.json [hunk 2]
```diff
@@ -511,13 +515,9 @@
...
   },
-  {
-    "id": "letter.initialize_proofing_help_text_tip_1",
-    "defaultMessage": "Tip: You can register at addresses outside of Sweden"
-  },
   {
     "id": "letter.initialize_proofing_help_text",
-    "defaultMessage": "Get a confirmation code sent by post to your registered address."
+    "defaultMessage": "The letter will contain a code that for security reasons expire in two weeks."
   },
```
Comment (enriquepablo) [inline i18n/src/i18n-messages.json]: I think it is either `expires` or `will expire`

File: i18n/l10n/sv.json [hunk 3]
```diff
@@ -108,62 +127,49 @@
...
-  "letter.confirm_info": "När du klickar \"acceptera\" kommer ett brev med en kod att skickas till din folkbokföringsadress. När du fått brevet så klickar du \"bekräfta med brev\" och skriver in koden för att bekräfta din identitet. Koden är giltig i två veckor.",
-  "letter.confirm_title": "Bekräfta identitet via brev till folkbokföringsadress",
-  "letter.initialize_proofing_help_text": "För att använda det här alternativet så måste du vänta på ett brev till din folkbokföringsadress.",
-  "letter.letter_button_text": "Bekräfta med brev",
+  "letter.button_text_code": "Skriv bekräftelsekoden här",
+  "letter.button_text_request": "via post",
+  "letter.initialize_proofing_help_text": "Brevet innehåller en bekräftelsekod som av säkehetsskäl går ut efter två veckor.",
   "letter.letter_sent_msg": "Ett brev med en kod har skickats till din folkbokföringsadress",
   "letter.lost_code": "Om du klickar på \"Skicka\"-knappen kommer en kod att skickas till din folkbokföringsadress.",
+  "letter.modal_confirm_info": "Om du accepterar att få etr brev hem måste du skriva in bekräfelsekoden här för att bevisa att personnumret är ditt. Av säkerhetsskäl går koden ut om två veckor.",
```
Comment (johanlundberg) [inline i18n/l10n/sv.json]: etr -> ett

File: i18n/l10n/sv.json [hunk 4]
```diff
@@ -108,62 +127,49 @@
...
-  "letter.verify_title": "Bekräfta identitet via brev till folkbokföringsadress",
+  "letter.verify_title": "Skriv in koden du fått hemskickad",
   "letter.wrong-code": "Den bekräftelsekod du angett stämmer inte. Var god försök igen",
-  "lmp.confirm-lookup-mobile": "Mobilabonnemang",
-  "lmp.initialize_proofing_help_text": "För att använda det här alternativet så måste ditt telefonabonnemang vara skrivet i ditt namn.",
+  "lmp.button_text_code": "Skriv bekräftelsekoden här",
+  "lmp.button_text_request": "via telefon",
+  "lmp.confirm_info": "Du kan acceptera en sökning i telefonregistret för att se om ditt telefonnummer finns lagrat tillsammans med ditt personnummer. Om dina uppgifter inte lagts till i registret av din mobiloperatör kommer vi inte kunna bevisa din identitet via telefonnummer.",
+  "lmp.initialize_proofing_help_text": "Registret med telefonnummer uppdateras av mobiloperatörerna och har inget krav på att innehålla alla nummer.",
+  "lmp.initialize_proofing_help_text_tip_1": "Tip: Registret med telefonnummer uppdateras av mobiloperatörerna och har inget krav på att innehålla alla nummer",
```
Comment (johanlundberg) [inline i18n/l10n/sv.json]: Tip: -> Tips:

File: i18n/l10n/sv.json [hunk 5]
```diff
@@ -248,73 +248,65 @@
...
+  "security.last_used": "Senast använt",
   "security.main_title": "Säkerhet",
-  "security.modal_info": "Är du säker på att du vill ta bort ditt eduID-konto? All information knuten till dig och ditt kontot kommer att tas bort.",
-  "security.modal_notes": "Av säkerhetsskäl så måste du logga in igen för att ta bort kontot.",
   "security.password_credential_type": "Lösenord",
   "security.remove": "Ta bort",
-  "security.u2f-describe-title": "Ange en beskrivning av Säkerhetsnyckeln du vill registrera.",
+  "security.second-factor": "Lägg till ett extra sätt, utöver email och lösenord, för att kunna identifiera dig så du är säker på att bara du har tillgång till ditt eduID.",
+  "security.security-key_title": "Gör ditt eduID säkrare",
+  "security.u2f-describe-title": "Ge säkerhetsnyckeln ett namn så du kan skilja dem åt om du lägger till fler.",
```
Comment (johanlundberg) [inline i18n/l10n/sv.json]: We should choose Säkerhetsnyckel or säkerhetsnyckel. I think we made the choice for capital S as that was the way Google does it. That doesn't mean we have to do the same choice as long as we are consistent :)

File: i18n/l10n/sv.json [hunk 6]
```diff
@@ -340,26 +332,41 @@
...
   "signup.recaptcha-not-verified": "Vi kunde inte verifiera att du är människa. Var vänlig försök igen.",
   "signup.registering-address-used": "E-postadressen används redan",
   "signup.registering-new": "E-postaddressen har registrerats",
   "signup.registering-resend-code": "Ett nytt verifieringsmeddelande har skickats",
   "signup.verification-resent": "Ett nytt verifieringsmeddelande har skickats",
-  "tl.make_primary": "SÄTT SOM PRIMÄR",
-  "tl.pending": "BEKRÄFTA",
-  "tl.primary": "PRIMÄR",
-  "tl.remove": "TA BORT",
+  "still-valid-code": "Vi kan bara skicka en kod var 5:e minut, var god vänta tills du ber om en ny kod.",
```
Comment (johanlundberg) [inline i18n/l10n/sv.json]: "var god vänta innan du ber om en ny kod", kanske?

File: i18n/l10n/sv.json [hunk 7]
```diff
@@ -340,26 +332,41 @@
...
   "used.forgot-password": "Glömt ditt lösenord?",
-  "used.reset-password": "ÅTERSTÄLL LÖSENORD",
-  "user-out-of-sync": "Användardata är ur synk. Var god försök igen."
-}
+  "used.reset-password": "Återställ lösenord",
+  "user-out-of-sync": "Användardata är ur synk. Var god försök igen.",
+  "verify-identity.connect_nin_title": "Välj en metod för att bevisa att personnumret är ditt",
+  "verify-identity.unverified_main_title": "Koppla din identitet till ditt eduID",
+  "verify-identity.verified_main_title": "Ditt eduID är redo att användas",
+  "verify-identity.vetting_freja_tagline": "För de som kan skapa Freja eID genom att besöka ett ombud i Sverige",
```
Comment (johanlundberg) [inline i18n/l10n/sv.json]: "För de som har eller kan skapa Freja eID", kanske?

File: i18n/l10n/sv.json [hunk 8]
```diff
@@ -5,37 +5,43 @@
...
   "No connection to authorization endpoint": "Anslutningen till auktoriseringsservern misslyckades. Var god försök senare",
   "Not a valid email address.": "Ogiltig e-postadress.",
   "Temporary technical problems": "Temporärt tekniskt fel, var god försök senare",
-  "account_linking.long_description": "Du kan länka andra konton till ditt eduID-konto.",
-  "account_linking.main_title": "Kontolänkning",
+  "accountId.accountId_display_title": "Unikt ID",
+  "accountId.long_description": "Detta är ett autogenererat unikt id för ditt eduID som du kan behöva ange när du ber om tekninsk support.",
+  "accountId.main_title": "Unikt ID",
+  "accountId.short_description": " ",
+  "account_linking.long_description": "Om du är forskare med ett ORCID iD, kan du länka det till ditt eduID och bara använda ett konto för att nå båda tjänsterna.",
```
[the remark this answers is not in the document]
Comment (njons) [reply i18n/l10n/sv.json]: :cry:

File: i18n/l10n/sv.json (hunk 5 above)
Comment (njons) [reply i18n/l10n/sv.json]: I did a quick google [here](https://support.google.com/accounts/answer/6103523?co=GENIE.Platform%3DAndroid&hl=en) (Eng) and [here](https://support.google.com/accounts/answer/6103523?co=GENIE.Platform%3DAndroid&hl=sv) (Swe) and could not find the capitalised use, so I will move forward with lowercase...of course do let me know if these are not the right sources.

**Pull request merged**